contrib/brl/bseg/boxm2/pro/processes/boxm2_load_scene_process.cxx

```diff
@@ -47,6 +47,7 @@ bool boxm2_load_scene_process(bprb_func_process& pro)
     scene = new boxm2_scene(scene_file);
   }
   catch(const std::ifstream::failure &e) {
+	  e.what();
```

Comment:
what does this accomplish?  (Did you mean to print it out?)

Reply:
I think it corrected a warning. I will go back and look at it again.

Reply:
It corrects the following warning:
1>C:\Users\cynthia.lucock\projects\vxl_fix_warnings\vxl\contrib\brl\bseg\boxm2\pro\processes\boxm2_load_scene_process.cxx(49): warning C4101: 'e': unreferenced local variable